talk/owt/sdk/conference/objc/RemoteMixedStreamObserverObjcImpl.h

```diff
@@ -13,7 +13,7 @@ class RemoteMixedStreamObserverObjcImpl : public RemoteMixedStreamObserver {
                                     id<OWTRemoteMixedStreamDelegate> delegate);
  protected:
   virtual void OnVideoLayoutChanged() override;
-  virtual void OnActiveInputChanged(std::string str) override;
+  virtual void OnActiveInputChanged(const std::string streamid) override;
```

Comment:
Suggest to use reference (`const std::string&`) to avoid copy. And `streamid`->`stream_id`.

Reply:
Done.